README.rst

```diff
@@ -5,8 +5,8 @@ cernopendata-client
 .. image:: https://img.shields.io/pypi/pyversions/cernopendata-client.svg
    :target: https://pypi.org/pypi/cernopendata-client
 
-.. image:: https://img.shields.io/travis/cernopendata/cernopendata-client.svg
-   :target: https://travis-ci.org/cernopendata/cernopendata-client
+.. image:: https://github.com/cernopendata/cernopendata-client/workflows/CI/badge.svg
+   :target: https://github.com/cernopendata/cernopendata-client/
```

Comment:
Ideally the target should bring you to last CI run details, not to the project github homepage...

Reply:
done 👍